### PR TITLE
Data: Add additional activity counts + consolidate generators

### PR DESCRIPTION
**Pull request by gblakeman**
Adds the following count types:

* Subscribers
* Comments
* Views
* Reactions
* Shares

Introduces a `generateInteractions` helper to consolidate a bunch of the type-specific generator functions into.

----

**Note:** The new types are set to “Beta” in Airtable and will need to be flipped to “Production” before merging.

### CODE REVIEW

File: src/Data.ts [hunk 1]
```diff
@@ -969,6 +768,99 @@ const generatePublishedFrequency = (): string => {
...
+ *
+ * @kind function
+ * @name generateInteractions
+ *
+ * @param {string} assignment A `RealishAssignment` string, restricted to comments, reacitons,
+ * shares, and views.
+ *
+ * @returns {string} The formatted number of attendees.
+ */
+const generateInteractions = (assignment: RealishAssignment): string => {
```
Comment (joeyyang) [inline src/Data.ts:784]: Feel free to punt this issue to another time, but is `RealishAssignment` always a `string`, like an `enum`? We could consider using the [enum](https://www.typescriptlang.org/docs/handbook/enums.html) feature, or use special casing (`REALISH_ASSIGNMENT`) to indicate that it's an enum.

File: src/Data.ts [hunk 2]
```diff
@@ -969,6 +768,99 @@ const generatePublishedFrequency = (): string => {
...
+      break;
+    case ASSIGNMENTS.reactions.id:
+      weightsArray = [1, 4, 8, 11, 12];
+      break;
+    case ASSIGNMENTS.shares.id:
+      weightsArray = [3, 5, 8, 11, 12];
+      break;
+    case ASSIGNMENTS.subscribers.id:
+      weightsArray = [2, 4, 9, 11, 12];
+      numArray = [1, 500, 1500, 32000, 180000];
```
Comment (joeyyang) [inline src/Data.ts:834]: I'd consider moving the weight info into a dict somewhere (`/constants.ts`?) and referencing it here so we can still follow the flow of the program. And maybe a discussion for another time, but I wonder if there's a way to simplify the number-generating scheme too - let me know if that sounds interesting.

**Review by lynnerang — APPROVED** [review]
LGTM!

**Review by joeyyang — APPROVED** [review]
A couple architecture questions since I'm looking at this with fresh(er) eyes, but otherwise this looks good!

File: src/Data.ts (hunk 2 above)
Comment (gblakeman) [reply src/Data.ts:834]: I’d like to keep the weight and number bounds info in `Data` for now. This generator function is the only place it’s used, so I think it makes sense to keep it where we’d want to adjust it.

I’m open to other changes _after_ I finish this round of work. This PR is a small part of a much larger refactor to all of the data/updater stuff.

**Pull request merged**